code/_hooks/events.dm

```diff
@@ -266,12 +288,11 @@
 	if(!length(event_handlers))
 		// This datum does not have any handler registered for this event_type.
 		return
-	. = NONE
 	for(var/key in event_handlers)
 		var/list/handler = event_handlers[key]
 		var/objRef = handler[EVENT_HANDLER_OBJREF_INDEX]
 		var/procName = handler[EVENT_HANDLER_PROCNAME_INDEX]
-		. |= CallAsync(objRef, procName, arguments)
```

Comment:
What's the difference here?

Reply:
Pretend this is on the new line rather than the old one. For some reason the comment button wouldn't show up on the new one

Reply:
`null |= list()` is a runtime error

`null = null | list()` is fine

Reply:
Ah, right. Might want to add a comment explaining that because it looks pretty arcane